mission-vision-values.md

```diff
@@ -116,3 +116,74 @@ to degrade gracefully as needed.
 OpenTelemetry users should not have to choose between high-quality telemetry and
 a performant application. High performance is a requirement for OpenTelemetry,
 and unexpected interference effects in the host application are unacceptable.
+
+## Community values &mdash; the principles that guide our interactions
+
+The OpenTelemetry project aims to be a welcoming place where new and existing
+members feel safe to respectfully share their opinions and disagreements. We
+want to attract a diverse group of people to collaborate with us, which means
+acknowledging that people come from different backgrounds and cultures. 
+
+There might be situations where community members act in a dubious manner. If
+you have seen or experienced unacceptable behavior or anything that would make
+our community less welcoming, please speak up! See [our code of
+conduct](https://github.com/open-telemetry/community/blob/main/code-of-conduct.md)
+for more information on how to report unacceptable behavior.
+
+While we want to encourage everyone to express themselves in their own way,
+there are some behaviors that we encourage you to adopt while interacting with
+other community members.
+
+### Act on behalf of the project
+
+It’s no secret that a good number of maintainers of the project are employed by
+companies with commercial interests in OpenTelemetry, especially vendors in the
+observability space. That said, we expect community members to act in the best
+interests of the project. Each member’s priorities can (and should!) align with
+those of their employers so that the relationship is beneficial to all parties,
+but when acting as a maintainer or contributor to the project, community members
```

Comment:
Suggestion: Change sentence to --> Each member’s priorities when acting as a maintainer or contributor for the project should represent the interests of the project and its community first before their other obligations (e.g. employer).